### PR TITLE
Add client-go auth plugin import

### LINKED ISSUE #24: no Auth Provider found for name "gcp"
```
$ kn event send --to broker:v1:default
Error: something unexpected actually has happened: no Auth Provider found for name "gcp"
```

/kind bug

xref: https://github.com/kubernetes/client-go/issues/242

### PR DESCRIPTION
**Pull request by navidshaikh**
# Changes
Add blank import for the client-go auth plugins,
which allows for kube configurations that use auth plugins
for client credentials to work correctly.

- :bug: Fixes #24

### CODE REVIEW

**Comment by rhuss** [comment]
Looks good to me, but the linter still croaks. Its probably not because of this PR but some deprecations happened in the meantime for some linter modules, so we could either merge the PR as is or also update the linter itself

**Comment by navidshaikh** [comment]
/assign @cardil 

@cardil : The CI is failing due to https://github.com/knative-sandbox/kn-plugin-event/issues/6 ?

**Review by rhuss — COMMENTED** [review]
+1 from my side, it mimics the same behaviour as the `kn` client itself, so for consistencies sake this would be very helpful.

wrt/ the failing test, also not sure what needs to be done to be fixed, but looks like some extra comments are require for `_` imports.

File: cmd/kn-event/cmd/root.go [hunk 1]
```diff
@@ -6,6 +6,7 @@ import (
 
 	"github.com/spf13/cobra"
 	"github.com/thediveo/enumflag"
+	_ "k8s.io/client-go/plugin/pkg/client/auth" // for kubeconfig auth plugins to work correctly see https://github.com/knative-sandbox/kn-plugin-event/issues/24
```
Comment (rhuss) [inline cmd/kn-event/cmd/root.go:9]: I think some comments are required for fixing the lint error `File is not `gci`-ed (gci)` ?

Comment (navidshaikh) [reply cmd/kn-event/cmd/root.go]: fixed

**Review by cardil — APPROVED** [review]
/lgtm

**Pull request merged**